### PR TITLE
Tidy version0.18.md

### PR DESCRIPTION
**Pull request by pmhayward**
Also update openj9_support.md for this release
and fix typo in xtrace .md (see #441)

Signed-off-by: Peter Hayward <pmhayward@uk.ibm.com>

### CODE REVIEW

**Comment by pmhayward** [comment]
Jenkins doc stage

File: docs/version0.18.md [hunk 1]
```diff
@@ -27,42 +27,41 @@
...
-- [`-Xmso` 1 MB minimum value on z/OS 64-bit](#-xmso-1-mb-minimum-value-on-zos-64-bit)
-- [New Java&trade; statistics monitoring (`jstat`) tool](#new-jstat-tool)
-- [-XX:+TransparentHugePage is enabled by default on more Linux systems](#-xxtransparenthugepage-is-enabled-by-default-on-more-linux-systems)
-- [Add new Xdump exit agent and ExitOnOutOfMemoryError option](#add-new-xdump-exit-agent-and-exitonoutofmemoryerror-option)
+- [`jextract` now available on macOS&reg; for OpenJDK version 8](#jextract-now-available-on-macos-for-openjdk-version-8)
+- [New shared-classes cache suboption to turn off timestamp checking](#new-shared-classes-cache-suboption-to-turn-off-timestamp-checking)
+- [`-Xmso` 1 MB minimum value on z/OS&reg; 64-bit](#-xmso-1-mb-minimum-value-on-zos-64-bit)
+- [`jstat`: new Java&trade; statistics monitoring tool](#jstat-new-java-statistics-monitoring-tool)
+- [`-XX:+TransparentHugePage` is enabled by default on more Linux&reg; systems](#-xxtransparenthugepage-is-enabled-by-default-on-more-linux-systems)
+- [New Xdump "exit" agent and `ExitOnOutOfMemoryError` option](#new-xdump-exit-agent-and-exitonoutofmemoryerror-option)
```
Comment (SueChaplain) [inline docs/version0.18.md]: We tend to refer to the agents as dump agents, not Xdump agents, so I think this item should be changed for consistency, including the section this links to. There is a confusing "See Xdump dump agents for more about agents." that would be better written something like "For more information about dump agents, see [`-Xdump`](https://eclipse.github.io/openj9-docs-staging/449/xdump/#dump-agents).

**Comment by pmhayward** [comment]
Jenkins doc stage

**Comment by pmhayward** [comment]
Jenkins doc stage

**Comment by pmhayward** [comment]
Jenkins doc stage

**Comment by SueChaplain** [comment]
LGTM

**Pull request merged**